test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -257,114 +242,82 @@ def api_client
       creds = GRPC::Core::CallCredentials.new(authentication.updater_proc)
       ssl_creds.compose(creds)
 
-      # Here we have obtained the creds, but for the mock, we will leave the
-      # channel insecure.
-      @grpc_stub.new(@mock_host, :this_channel_is_insecure)
+      @grpc_stub
     end
   end
 
   # GRPC logging mock that successfully logs the records.
-  class GRPCLoggingMockService < Google::Logging::V2::LoggingServiceV2::Service
+  class GRPCLoggingMockService <
+      Google::Cloud::Logging::V2::LoggingServiceV2Client
     def initialize(requests_received)
       super()
       @requests_received = requests_received
     end
 
-    def write_log_entries(request, _call)
+    def write_log_entries(entries,
```

Comment:
This whole signature should fit on one line.

Reply:
Changed.